sklearn/utils/estimator_checks.py

```diff
@@ -2338,7 +2338,7 @@ def check_classifiers_regression_target(name, estimator_orig):
 
 @ignore_warnings(category=(DeprecationWarning, FutureWarning))
 def check_estimator_sparse_dense(name, estimator_orig):
-    rng = np.random.RandomState(42)
+    rng = np.random.RandomState(52)
```

Comment:
@agramfort I had the same problem of random seed here for `AdaBoostClassifier`: (tested at the end of `test_check_estimator`) 
this random seed fixed it